api/pom.xml

```diff
@@ -35,6 +35,10 @@
          <groupId>org.apache.commons</groupId>
          <artifactId>commons-lang3</artifactId>
       </dependency>
+      <dependency>
```

Comment:
Is this change related to updating apache velocity?

Reply:
@dkayiwa yes apache velocity 2.0 necessitated also dependency on commons-lang3